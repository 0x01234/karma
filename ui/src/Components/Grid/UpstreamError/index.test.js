import React from "react";
import renderer from "react-test-renderer";

import { UpstreamError } from ".";

describe("<UpstreamError />", () => {
  it("matches snapshot", () => {
    const tree = renderer
      .create(<UpstreamError name="foo" message="bar" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
