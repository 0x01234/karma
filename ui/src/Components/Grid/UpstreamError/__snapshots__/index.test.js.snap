// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UpstreamError /> matches snapshot 1`] = `
<div
  className="alert alert-danger text-center"
  role="alert"
>
  <h4
    className="alert-heading mb-0"
  >
    Alertmanager 
    <span
      className="font-weight-bold"
    >
      foo
    </span>
     raised an error: 
    bar
  </h4>
</div>
`;
